modules/snooker/src/test/ts/browser/CloneFormatsTest.ts

```diff
@@ -12,7 +12,7 @@ UnitTest.test('CloneFormatsTest', () => {
   const noCloneTableFill = TableFill.cellOperations(Fun.noop, doc, noCloneFormats);
 
   const cellElement = SugarElement.fromTag('td');
-  const cellContent = SugarElement.fromHtml('<strong contenteditable="false"><em>stuff</em></strong>');
+  const cellContent = SugarElement.fromHtml('<strong><em contenteditable="false">stuff</em></strong>');
```

Comment:
Should we keep the old test as well where it was checking that the outer format had contenteditable=false on it? If so, what should its expected value be?